policyeval/execute.go

```diff
@@ -253,6 +253,10 @@ func (pe *PolicyEvaluator) redactUserSynapse(ctx context.Context, userID id.User
 }
```

Comment:
These changes were merged in #22, could you remove this file from the commit?

Reply:
After way too much git fiddling, done lol